engine/access/apiproxy/access_api_proxy.go

```diff
@@ -136,50 +135,49 @@ func (h *FlowAccessAPIForwarder) faultTolerantClient() (access.AccessAPIClient,
 // Ping pings the service. It is special in the sense that it responds successful,
 // only if all underlying services are ready.
 func (h *FlowAccessAPIRouter) Ping(context context.Context, req *access.PingRequest) (*access.PingResponse, error) {
-	res, err := h.AccessAPIServer.Ping(context, req)
-	h.log("observer", "Ping", err)
-	return res, err
+	h.log("observer", "Ping", nil)
+	return &access.PingResponse{}, nil
 }
```

Comment:
`Ping` is now implemented here because it doesn't make sense to forward upstream and the `protocol.API` interface does not have a `Ping` method. I think adding `Ping` to the  `protocol.API` also makes sense

Reply:
we don't need to have `Ping` in protocol either